docs/quick-start.md

```diff
@@ -65,7 +65,16 @@ result = invoke(dag, params={"seed": 1, "exponent": 2})
 print(f"The final result was {result}")
 ```
 
-After this, you should see the results of the DAG printed to your screen.
+After this, you should see the results of the DAG printed to your screen. 
+
+Equivalently, because the parameter `exponent` has a default value of 2, you can omit passing the value 2 when invoking the DAG:
+
+```python
+from dagger.runtime.local import invoke
+
+result = invoke(dag, params={"seed": 1})
+print(f"The final result was {result}")
+```
```

Comment:
This code is wrong. Running this results in an exception:

> ValueError: The parameters supplied to this node were supposed to contain the following parameters: ['exponent', 'seed']. However, only the following parameters were actually supplied: ['seed']. We are missing: ['exponent'].

The problem is that the default value is in a task, not in the DAG. I think I'd put the default in the DAG. It's more illustrative of what we're trying to do.

Reply:
true, done!